docs/reference/fields.txt

```diff
@@ -897,6 +897,17 @@ can use in our model class as follows:
     field :location, type: Point
   end
 
+You may optionally declare a symbol mapping for the new field type in an initializer:
+
+.. code-block:: ruby
+
+  # in /config/initializers/mongoid_custom_fields.rb
+
+  Mongoid::Fields.configure do
+    type :point, Point
```

Comment:
`type` is ambiguous here, for example if we wanted to require the allowed set of types to be defined before use that definition could potentially need `type`. What do you think about `alias` instead?

Reply:
`alias` is a ruby keyword, so should avoid it. how about `define_type` or `field_type`?

Reply:
You're right, I overlooked that. This is not a type definition but an alias definition, so what about `type_alias`?

Reply:
The word `alias` implies like-for-like, e.g. symbol-for-symbol `type_alias :bool, :boolean`.

I think `define_type` is best.

Reply:
Changed to `define_type`